release-notes.en.md

```diff
@@ -2,6 +2,12 @@
 
 [日本語](./release-notes.md)
 
+## [Version 2.5.0](https://github.com/skyway/skyway-android-sdk/releases/tag/v2.5.0) - 2021-02-02
+
+### Changed
+
+- Changed `dataChannel.send()` interval so that multiple data can be transmitted without delay even when continuously transmitting at an intervals of less than 100 msec.
```

Comment:
```suggestion
- Changed `dataChannel.send()` interval so that multiple data can be transmitted without delay even when those data are continuously transmitted at an intervals of less than 100 msec.
```